src/wfinfo.c

```diff
@@ -1756,22 +1756,10 @@ NetLoad(VOID)
          //
          bNetShareLoad = TRUE;
 
-      } else {
-Fail:
-         //
-         // Disable the share buttons/menus
-         // Since WNetStat(NS_SHAREDLG) ret'd true, then we added the
-         // buttons.  Mistake.  Disable them now.
-         //
-         PostMessage(hwndToolbar, TB_ENABLEBUTTON, IDM_SHAREAS, FALSE);
          PostMessage(hwndToolbar, TB_ENABLEBUTTON, IDM_STOPSHARE, FALSE);

```

Comment:
Shouldn't this be removed? If I'm reading this right, the toolbar will never have an IDM_STOPSHARE item so this will always fail?

Reply:
Dupe comment